app/views/admin/users/index.html.erb

```diff
@@ -1,10 +1,16 @@
-<h1><%= t(".title") %></h1>
-<% @users.each do |user| %>
-  <ul id="user_<%= user.id %>">
-    <li><%= t(".name") %>: <%= user.name %></li>
-    <li><%= t(".email") %>: <%= user.email %></li>
-    <li><%= t(".admin") %>: <%= user.admin? %></li>
-    <li><%= link_to t(".links.delete"), admin_user_path(user), method: :delete %>
-      | <%= link_to t(".links.edit"), edit_admin_user_path(user) %></li>
-  </ul>
-<% end %>
+<table>
+  <h3 class="table_align"><%= t(".title") %></h3>
+  <th><%= t(".name") %></th>
+  <th><%= t(".email") %></th>
+  <th><%= t(".admin") %></th>
+  <th></th>
+  <% @users.each do |user| %>
+    <tr id="user_<%= user.id %>">
+      <td><%= user.name %></td>
+      <td><%= user.email %></td>
+      <td><%= user.admin? %></td>
+      <td><%= link_to t(".links.delete"), admin_user_path(user), method: :delete %>
+        <%= link_to t(".links.edit"), edit_admin_user_path(user) %></td>
```

Comment:
I think it can be nicer in this way:
``` 
 <td>
   <%...%>
   <%..%>
</td>
```

Reply:
I may leave it as is as it is already a long file, thanks though!

Reply:
Using different lines for each html element, makes it easier to read, and also easier later on with git diffs.

Reply:
Done in sha: a2fa585